pkg/k8sutil/create.go

```diff
@@ -227,3 +230,93 @@ func createNamespace(ns Namespace, nsclient corev1typed.NamespaceInterface) erro
 func ListNamespaces(nsclient corev1typed.NamespaceInterface) (*v1.NamespaceList, error) {
 	return nsclient.List(context.TODO(), metav1.ListOptions{})
 }
+
+// ObjectMetadata uniquely identifies any object in the list of YAML manifests.
```

Comment:
With this changes, we should mark `LoadManifests` as deprecated.

Reply:
How do we mark something as deprecated right now?

Reply:
I think just mentioning in Function description that other one should be used and that this one is deprecated would be sufficient. We don't use deprecated special keyword, as it triggers a linter then.